### PR TITLE
Feat 13 navigation desktop

### PR DESCRIPTION
**Pull request by SinchanaMS**
Created the Navigation Component.

1. Top Navigation
2. Sidebar

Preview:
1. Top Navigation:
image

2. Sidebar: 
image

### CODE REVIEW

**Review by srejitk — APPROVED** [review]
No changes suggested. Looks good. 
Liked the intuitive class names.

**Review by saurabhchirde — APPROVED** [review]
Couldn't find any issue, 
Looks good to merge.

File: stylesheets/documentation.css [hunk 1]
```diff
@@ -455,6 +454,50 @@ pre {
...
+/* Navigation */
+.top-nav-wrapper{
+    background: var(--SIMPLE-ALERTS-SECTION)
+}
+
+.top-nav{
+    padding: 1rem;
+    background: var(--SECONDARY-COLOR);
+    border-radius: 4px;
+    width: 100%;
```
Comment (tarunsankhla) [inline stylesheets/documentation.css:466]: I think max width and width will do the same thing here

**Review by tarunsankhla — APPROVED** [review]
Over all everything looks good just add comments on the html part

File: stylesheets/documentation.css (hunk 1 above)
Comment (SinchanaMS) [reply stylesheets/documentation.css:466]: Thank you, I'll check that.

**Comment by SinchanaMS** [comment]
Thank you for the reviews, merging to temp-dev.

**Pull request merged**